packages/core/admin/admin/src/components/NpsSurvey/tests/index.test.js

```diff
@@ -4,9 +4,26 @@ import { lightTheme, ThemeProvider } from '@strapi/design-system';
 import { render, screen, waitFor, fireEvent, act } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { IntlProvider } from 'react-intl';
+import { QueryClient, QueryClientProvider } from 'react-query';
 
 import NpsSurvey from '..';
 
+const toggleNotification = jest.fn();
+
+jest.mock('@strapi/helper-plugin', () => ({
+  ...jest.requireActual('@strapi/helper-plugin'),
+  auth: {
+    getUserInfo: jest.fn(() => ({
+      email: 'john@doe.com',
+    })),
+  },
+  useNotification: jest.fn().mockImplementation(() => toggleNotification),
+}));
+
+global.fetch = jest.fn(() => ({
+  ok: true,
+}));
```

Comment:
What do you think about mocking the API using msw and write a very simple test for it?